content/en/docs/components/spark-operator/who-is-using.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Who is Using
```

Comment:
I would suggest to create ADOPTERS.md file in the Spark Operator repo: https://github.com/kubeflow/spark-operator and move this doc there.
Similar to this one: https://github.com/kubeflow/katib/blob/master/ADOPTERS.md
WDYT @ChenYi015 @vara-bonthu @yuchaoran2011 ?

Reply:
@andreyvelich Thanks for your advice, already removed it from this PR.